adapters/nextmillennium/nextmillennium.go

```diff
@@ -99,12 +99,29 @@ func (adapter *adapter) buildAdapterRequest(prebidBidRequest *openrtb2.BidReques
 }
 
 func createBidRequest(prebidBidRequest *openrtb2.BidRequest, params *openrtb_ext.ImpExtNextMillennium) *NextMillenniumBidRequest {
+	placementID := params.PlacementID
 	bidRequest := NextMillenniumBidRequest{
 		ID:   prebidBidRequest.ID,
 		Test: uint8(prebidBidRequest.Test),
 	}
-	bidRequest.Ext.Prebid.StoredRequest.ID = params.PlacementID
 
+	if params.GroupID != nil {
+		domain := ""
+		if prebidBidRequest.Site != nil && prebidBidRequest.Site.Domain != "" {
+			domain = prebidBidRequest.Site.Domain
+		}
+		if prebidBidRequest.App != nil && prebidBidRequest.App.Domain != "" {
+			domain = prebidBidRequest.App.Domain
+		}
+		placementID = fmt.Sprintf(
+			"g%s;%dx%d;%s", *params.GroupID,
+			*prebidBidRequest.Imp[0].Banner.W,
```

Comment:
Please add validation to make sure `Imp[0].Banner` is not `nil` before accessing the `W` and `H` fields

Reply:
@mansinahar Done!